lib/timeline/component/ItemSet.js

```diff
@@ -1662,21 +1685,19 @@ ItemSet.prototype._onGroupClick = function (event) {
 
   if (!group || !group.nestedGroups) return;
 
-  var groupsData = this.groupsData;
-  if (this.groupsData instanceof DataView) {
-    groupsData = this.groupsData.getDataSet()
-  }
+  var groupsData = this.groupsData.getDataSet();
 
-  group.showNested = !group.showNested;
+  var nestingGroup = groupsData.get(group.groupId)
+  nestingGroup.showNested = !nestingGroup.showNested;
```

Comment:
When a nesting group is initialy loaded as 'open' (the implicit "showNested: true"), showNested is undefined. Negating it turns it to true which means expand, correct? This causes that an open loaded nesting group needs to be clicked twice. Adding this the line before should do the trick:

`if (nestingGroup.showNested == undefined) nestingGroup.showNested = true;`